octodns/provider/azuredns.py

```diff
@@ -17,5 +17,7 @@
 except ModuleNotFoundError:
     logger.exception('AzureProvider has been moved into a seperate module, '
                      'octodns_azure is now required. Provider class should '
-                     'be updated to octodns_azure.AzureProvider')
+                     'be updated to octodns_azure.AzureProvider. See '
+                     'https://github.com/octodns/octodns/README.md#updating-'
```

Comment:
```suggestion
                     'https://github.com/octodns/octodns#updating-'
```

`/README.md` isn't a valid GitHub file path. It has to be `/blob/<ref>/README.md` or since the root renders the readme, it can be just at the root of the repo:

https://github.com/octodns/octodns/#updating-to-use-extracted-providers